templates/patterns/modal/pattern-modal.html.twig

```diff
@@ -4,6 +4,7 @@
  * Modal pattern.
  */
 #}
+{{ attach_library('oe_bootstrap_theme/accessible_toggle') }}
```

Comment:
what about language list modal for example, and there could be other modals that I miss, rather make this part of the bcl library so it gets loaded on every page and acts wherever needed.